src/main/java/com/example/home_work_4/StringTotal.java

```diff
@@ -0,0 +1,393 @@
+package com.example.home_work_4;
+
+import java.util.Scanner;
+
+public class StringTotal {
+
+    public static void main(String[] args) {
+        //String1_1
+        String s1 = "<<>>";
+        Scanner scanner = new Scanner(System.in);
+        System.out.print("Enter your word: ");
+        String yourWord = scanner.nextLine();
+        System.out.println(s1.substring(0,2)+yourWord+s1.substring(2));
+
+        //String1_2
+        Scanner scanner = new Scanner(System.in);
```

Comment:
Unfortunately, your code doesn't compile. Please refactor it and make it compile.